infra/integration-image/Dockerfile.test

```diff
@@ -86,4 +89,7 @@ RUN wget "https://raw.githubusercontent.com/docker/docker/${DIND_COMMIT}/hack/di
 COPY dockerd-entrypoint.sh /usr/local/bin/dockerd-entrypoint.sh
 Run chmod +x /usr/local/bin/dockerd-entrypoint.sh
 
+RUN git clone https://github.com/andrewtchin/ca-test.git /opt/vmware/ca-test
```

Comment:
because i wrote them for personal use, mike was ok with pulling them in like this